dev/ansible/05_build_docker_image.yml

```diff
@@ -15,6 +16,7 @@
       args:
         TEST_DOWNLOAD_BUILD_ARGUMENT: test-arg
     source: build
+  when: lookup('env', 'build_docker_image') | bool or false
```

Comment:
What means `bool or false`?

Reply:
This part make look up and cast result to Boolean `lookup('env', 'build_docker_image') | bool`, part with `or false` is like default value, equal to `|| false`

Reply:
Aha! Thanks for the clarification.